tests/unit/json/pull_file_patch_example

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Slight nit: Can we make this file name more accurate, e.g., `pull_file_example_without_patch` or `pull_file_missing_patch_example`?